.github/workflows/ci.yml

```diff
@@ -26,9 +26,9 @@ jobs:
 
       - name: Install
         run: |
-          wget https://raw.githubusercontent.com/api7/apisix-build-tools/master/build-apisix-openresty.sh
-          chmod +x build-apisix-openresty.sh
-          OR_PREFIX=$OPENRESTY_PREFIX ./build-apisix-openresty.sh latest
+          wget https://raw.githubusercontent.com/api7/apisix-build-tools/master/build-apisix-base.sh
```

Comment:
@spacewander how about import the versioned script via `https://raw.githubusercontent.com/api7/apisix-build-tools/v2.2.1/build-apisix-openresty.sh`? The code in the master branch is risky. Looking forward to your feedback.